blazor/dropdown-list/selection.md

```diff
@@ -0,0 +1,147 @@
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/change-the-selected-value.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with pre-select value before](./images/selection/blazor_dropdown_changing-selected-value.gif)
+
+## Preselect Value with Index
+
+You can bind pre-selected value to the component also using [@bind-Index](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Index) property. It binds the respective value present in the binded index.
+
+> It will dependent on [SortOrder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SortOrder.html) type. It will bind the value to the component with sorted data if the corresponding property is defined.
+The below sample demonstrates how to bind the index on initial rendering.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/preselect-value-index.razor %}
+
+{% endhighlight %}
+
+The output will be as follows,
+
+![Blazor DropDownList with bind-index](./images/selection/blazor_dropdown_preselect-value-index.png)
+
+## Get Selected Item by Value
+
+You can get all the datasource items belongs to the value selected in the component by using [GetDataByValue](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_GetDataByValue__0_) method. It will returns the items with the return type of TValue mapped to the component. 
+
+The below sample demonstrates usage `GetDataByValue` method,
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/get-selected-item-by-value.razor %}
+
+{% endhighlight %}
+
+## Focus the Next Component on Selection
+
+You can focus the component programmatically using [FocusAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_FocusAsync) public method. It will set focus instantly to the DropDownList component when invoking it. 
+
+Also switch focus from one component to another using action based events. In the below sample demonstration, the focus will move from dropdown A to dropdown B if the value is selected in the dropdown A. Hence we have invoked the respective focusing method of B component inside A component's Closed event.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/focus-next-component.razor %}
+
+{% endhighlight %}
+
+## Programmatically Trigger onChange Event
+
+You can trigger the ValueChange event manually by using the instance (taken from @ref property) of the **DropDownListEvents** tag helper. In the below example, we have invoked the ValueChange event inside **Created** event handle. As per the below code it will trigger once the component is created/rendered in the page.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/trigger-change-event.razor %}
+
+{% endhighlight %}
+

```

Comment:
Add "##Events" and then move the respective events as sub content "### ValueChange Event"